docs/cli.md

```diff
@@ -2,22 +2,36 @@
 
 ## Overview
 
-The CLI is the primary API you'll use when setting up Lighthouse CI. Install the CLI globally to try it out locally.
+The CLI is the primary API you'll use when setting up Lighthouse CI. To install:
 
 ```bash
 npm install -g @lhci/cli
 ```
 
-or
-
-```bash
-yarn global add @lhci/cli
```

Comment:
😆 the original author here is gonna be so disappointed. they originally added duplicate copies of every single usage of `npm` and I talked them into this instead :)